[2022-12-04 10:48:10,127] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-12-04T07:04:47.742537+00:00 [queued]>
[2022-12-04 10:48:10,166] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-12-04T07:04:47.742537+00:00 [queued]>
[2022-12-04 10:48:10,169] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 10:48:10,169] {taskinstance.py:1239} INFO - Starting attempt 6 of 7
[2022-12-04 10:48:10,169] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 10:48:10,225] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-12-04 07:04:47.742537+00:00
[2022-12-04 10:48:10,246] {standard_task_runner.py:52} INFO - Started process 166 to run task
[2022-12-04 10:48:10,284] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'download_dataset_task', 'manual__2022-12-04T07:04:47.742537+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp_niaowx7', '--error-file', '/tmp/tmpqzg2yhdb']
[2022-12-04 10:48:10,293] {standard_task_runner.py:77} INFO - Job 28: Subtask download_dataset_task
[2022-12-04 10:48:10,512] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-12-04T07:04:47.742537+00:00 [running]> on host deb8ed2c55fa
[2022-12-04 10:48:10,730] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-12-04T07:04:47.742537+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-04T07:04:47.742537+00:00
[2022-12-04 10:48:10,734] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-12-04 10:48:10,735] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSL https://d37ci6vzurychx.cloudfront.net/misc/taxi+_zone_lookup.csv > /opt/***/taxi+_zone_lookup.csv']
[2022-12-04 10:48:10,778] {subprocess.py:85} INFO - Output:
[2022-12-04 10:48:11,047] {subprocess.py:93} INFO - Command exited with return code 0
[2022-12-04 10:48:11,217] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=download_dataset_task, execution_date=20221204T070447, start_date=20221204T104810, end_date=20221204T104811
[2022-12-04 10:48:11,341] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-12-04 10:48:11,745] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
